### DISCUSSION TITLE
Movable dialog/modal examples

### DISCUSSION THREAD

**Discussion by mfulton26** [mantinedev/mantine#6556] [Q&A]
I see that Mantine has `useMove` and `Dialog` as well as `Modal` but despite trying various things I can't get `useMove()` to work well with `Dialog` or `Modal`. I am using `react-draggable` at the moment instead (with `Dialog`). Some examples on how to do this on the `Dialog`/`Modal` docs pages and/or on the `useMove` doc page would be greatly appreciated.

**Comment by rtivital** [comment]
use-move is deigned for custom sliders. For your use case, it is mot suitable to use react-draggable.
